src/PKSim.Core/Services/MoleculesAndReactionsCreator.cs

```diff
@@ -215,13 +215,18 @@ private void addMolecule(IndividualMolecule individualMolecule)
 
          addMoleculeToBuildingBlock(molecule, compoundProperties: null);
 
-         //Update protein builder parameters with the parameter ids
+         //Update protein builder parameters with the parameter ids and values
          _parameterSetUpdater.UpdateValuesByName(individualMolecule, molecule.Parameters);
 
          //Update the building block ids
          _parameterIdUpdater.UpdateBuildingBlockId(molecule.Parameters, _individual);
+
+         //since parameter value were updated from default, some IsDefault will be reset to false
+         _parameterIdUpdater.ResetParameterIsDefaultState(molecule, individualMolecule.GetChildren<IParameter>());
```

Comment:
this was missing and I realize that all global parameter are always exported in an simulation